python/sample.py

```diff
@@ -1,22 +0,0 @@
-import numpy as np
```

Comment:
I think it is good to have a very basic super clear example of using iris from python. this was an example